rllib/utils/actor_manager.py

```diff
@@ -467,15 +471,16 @@ def __fetch_result(
         for r in ready:
             # Find the corresponding actor ID for this remote call.
             actor_id = remote_actor_ids[remote_calls.index(r)]
+            tag = tags[remote_calls.index(r)]
```

Comment:
oh I think the reason we did this is because of the way that `foreach_actor` is written. I'm not sure if ti is modifying `self.__in_flight_req_to_actor_id`

Reply:
resolved offline